### PR TITLE
Add doc on support of "rdl:targetSameWindow=true"

### PR DESCRIPTION
**Pull request by lishibo**
This is a new url parameter support released in 2020.6 train. Updating doc reflecting this.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@lishibo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by lishibo** [comment]
@maggiesMSFT, I'm not sure if I'm adding the link correctly. Please let me know what's the proper way of referencing other doc pages.

File: powerbi-docs/paginated-reports/report-builder-url-parameters.md [hunk 1]
```diff
@@ -165,6 +165,9 @@ XML:
 - rdl:FileExtension=string
 - rdl:Schema=true/false
 
+**Open hyperlink in same browser window**
+You can append 'rdl:targetSameWindow=true' to the hyperlink url in your report, and it'll make Power BI to open this hyperlink in same browser window. For information on how to add hyperlink in your report, see [Add a Hyperlink to a URL (Report Builder and SSRS)](https://docs.microsoft.com/en-us/sql/reporting-services/report-design/add-a-hyperlink-to-a-url-report-builder-and-ssrs?redirectedfrom=MSDN&view=sql-server-ver15)
```
Comment (maggiesMSFT) [inline powerbi-docs/paginated-reports/report-builder-url-parameters.md:169]:
```suggestion
You can append 'rdl:targetSameWindow=true' to the hyperlink URL in your report to make Power BI to open this hyperlink in the same browser window. For information on adding hyperlinks to a report, see [Add a hyperlink to a URL](https://docs.microsoft.com/sql/reporting-services/report-design/add-a-hyperlink-to-a-url-report-builder-and-ssrs) in the SQL Server Reporting Services documentation.
```

**Comment by PRMerger16** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**